modules/iam/output.tf

```diff
@@ -9,3 +9,13 @@ output "iam_emr_service_role" {
 output "iam_emr_autoscaling_role" {
   value = "${aws_iam_role.segment_emr_autoscaling_role.name}"
 }
+
+output "data_lake_iam_role_arn" {
+  value = "${aws_iam_role.segment_data_lake_iam_role.arn}"
+}
+
+output "iam_emr_instance_profile_arn" {
```

Comment:
Rename this to `iam_emr_instance_profile_role_arn`

Reply:
done